src/main/java/io/strimzi/kafka/quotas/StaticQuotaCallback.java

```diff
@@ -180,13 +178,37 @@ public Long value() {
         });
     }
 
-    private void updateUsedStorage(Long newValue) {
-        var oldValue = storageUsed.getAndSet(newValue);
-        if (oldValue != newValue) {
+    //@VisiableForTesting
+    void calculateQuotaFactor(Map<String, Long> usagePerDisk) {
```

Comment:
We often do this to make it visible for testing:

```suggestion
    /* test */ void calculateQuotaFactor(Map<String, Long> usagePerDisk) {
```